apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/KeyStoreController.java

```diff
@@ -204,7 +200,7 @@ public Response downloadKeyStore(@FormParam("account") String account,
             long accountId = HttpParameterParserUtil.getAccountId(account, "account", true);
```

Comment:
HttpParameterParserUtil is deprecated, we should use RestParametersParser instead of this one.